ocsp/responder.go

```diff
@@ -82,15 +121,20 @@ func NewSourceFromFile(responseFile string) (Source, error) {
 // Source of OCSP responses.
 type Responder struct {
 	Source Source
+	eTags  *eTagPool
 	clk    clock.Clock
 }
 
 // NewResponder instantiates a Responder with the give Source.
-func NewResponder(source Source) *Responder {
-	return &Responder{
+func NewResponder(source Source, useETags bool) *Responder {
+	rs := &Responder{
 		Source: source,
 		clk:    clock.Default(),
 	}
+	if useETags {
+		rs.eTags = &eTagPool{make(map[string]time.Time), rs.clk, new(sync.RWMutex)}
```

Comment:
It might be better to add a `newETagPool` function that handles setting up a new `eTagPool`; it would be cleaner use field names and easier to update in one place if the definition of an `eTagPool` changes.